setup/setup.sh

```
@@ -0,0 +1,149 @@
+else
+  echo "Virtualization is not enabled. Please enable it in BIOS settings."
+  echo "Without virtualization Docker may not work properly."
+  exit 1
+fi
+
+# Check if Docker is installed
+if command_exists docker; then
+  echo "Docker is installed on this machine."
+else
+  echo "Docker is not installed."
+  if [[ "$use_pkgmgr" == "yes" ]]; then
```

Shellcheck (0.10.0)

[warning] use_pkgmgr is referenced but not assigned.

(SC2154)